### PR TITLE
crypt: log hash ok on upload

### PR DESCRIPTION
**Pull request by albertony**
#### What is the purpose of this change?

When copying files into non-crypt remotes:

    DEBUG : zip/kdbx.20210314.185330.7z: MD5 = 814e77ae9ae377c3201193c858da46d7 OK
    INFO  : zip/kdbx.20210314.185330.7z: Copied (new)

But when copying file into crypt remote, there is no info about integrity check even if it is performed (unless it fails and it will be reported as part of "corrupted on transfer" error):

    INFO : 01.kdbx: Copied (new)

This change adds the same debug log for crypt remotes.

#### Was the change discussed in an issue or in the forum before?

https://forum.rclone.org/t/local-copy-operation-fails-with-handle-is-invalid/22880/12

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [X] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by ivandeex** [comment]
LGTM

**Comment by albertony** [comment]
> LGTM

Thanks @ivandeex. What do you think: Could/should I just merge simple PR like this, at least after your's, or other devs "approval", or best always wait for a final approval from da man?

**Comment by ivandeex** [comment]
I am usually waiting. Can't recommend anything.

**Review by ncw — APPROVED** [review]
That looks fine to me - please merge :-)

**Comment by ncw** [comment]
@albertony wrote:
> > LGTM
> 
> Thanks @ivandeex. What do you think: Could/should I just merge simple PR like this, at least after your's, or other devs "approval", or best always wait for a final approval from da man?

:-)

I think @ivandeex approval for this is fine for such a simple PR :-)

In general, if you merge stuff which breaks things, you get to fix it!

**Pull request merged**
